analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -597,11 +597,12 @@ open class Npm(
         )
 
         fun mapLinesToOrtIssues(prefix: String, severity: Severity) {
-            val issueLines = lines.takeWhile { it.startsWith(prefix) }.mapNotNull { line ->
-                line.removePrefix(prefix).let {
-                    commonSecondaryPrefixes.fold(it) { remainder, prefix -> remainder.removePrefix(prefix) }
-                }.takeUnless { it.isBlank() }
-            }
+            val issueLines = lines.dropWhile { !it.startsWith(prefix) }
+                .takeWhile { it.startsWith(prefix) }.mapNotNull { line ->
```

Comment:
Is `.takeWhile { it.startsWith(prefix) }` now redudant?

Reply:
I don't think so. This ensures that output with a different log level coming afterwards is not added to the issue description.

Reply:
Oh, I hadn't understood the code before correctly. Hope I now have....

So, after your change the code keeps the assumption that the output is grouped by prefix.
e.g. all warnings are next to one another and all errors are next to one another.

Should this assumption be kept? (If not, maybe then my proposal here https://github.com/oss-review-toolkit/ort/pull/6418/commits/1d592a67c96c788f92e16d5fc4a8f6cc2377da00#r1091792687 might make sense)

Reply:
From what I have seen so far, it seems to be reasonable that the output is grouped by prefix; so I would keep it.